src/components/Games/Number/NumberMain.js

```diff
@@ -56,6 +56,11 @@ const GuessButton = styled.button`
   }
 `;
 
+const ButtonContainer = styled.div`
+{display: flex; gap: 5px;};
```

Comment:
This somehow works but you don't need the `{}` just do `display: flex;` and `gap: 5px;` on a new line.
Same in SequenceMain.js